enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper/PublicComponentWrapper.tsx

```diff
@@ -32,5 +33,9 @@ export const PublicComponentWrapper = ({
     content = <SdkError message={loginStatus.error.message} />;
   }
 
-  return <PublicComponentStylesWrapper>{content}</PublicComponentStylesWrapper>;
+  return (
+    <PublicComponentStylesWrapper {...props}>
```

Comment:
From https://github.com/metabase/metabase/blob/a471151fdbaf06fe4c8d16f79f2db59143824999/enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboard.tsx#L82

It seems we only pass 3 more props, I'd rather define those 3 props directly, instead of allowing passing anything which could make it harder to know what props we are really using.

I'd say unless we expect this to be a public API i.e. letting SDK users to customize the wrapped div component, we shouldn't be spread props.